cmd/itzo-launcher/itzo-launcher.go

```diff
@@ -39,8 +38,21 @@ var (
 var (
 	version    = flag.Bool("version", false, "print version and exit")
 	itzoLogDir = flag.String("itzo-log-dir", "/var/log/itzo", "directory for itzo.log")
+	ItzoDefaultURL = "https://itzo-kip-download.s3.amazonaws.com"
+	ItzoDefaultVersion = "latest"
 )
 
+func init()  {
+	switch runtime.GOARCH {
+	case "arm64":
+		ItzoDefaultURL = "https://itzo-kip-download.s3.amazonaws.com"
+		ItzoDefaultVersion = "arm-latest"
+	default:
+		ItzoDefaultURL = "https://itzo-kip-download.s3.amazonaws.com"
+		ItzoDefaultVersion = "latest"
+	}
+}
```

Comment:
Using global variables isn't great. Instead I would define two functions: GetItzoURL & GetItzoVersion to return to appropriate values based on runtime.GOARCH.

Reply:
I believe refactoring EnsureItzo() will make things cleaner & easier.